core/config/initializers/assets.rb

```diff
@@ -1,6 +1,27 @@
+# These precompile sets are split for readablity
+# scripts
+ Rails.application.config.assets.precompile += %w(
```

Comment:
This is now the place where someone following the Rails recommendations would expect to find them.
I'm not sure why "hidden in engine.rb" is a good idea.